packages/fx-core/templates/plugins/resource/aad/auth/tab/README.txt

```diff
@@ -0,0 +1,46 @@
+Enable single sign-on for tab applications
+-------------------------
+Microsoft Teams provides a mechanism by which an application can obtain the signed-in Teams user token to access Microsoft Graph (and other APIs).
+Teams Toolkit facilitates this interaction by abstracting some of the Azure Active Directory flows and integrations behind some simple, high level APIs.
+This enables you to add single sign-on (SSO) features easily to your Teams application.
+
+Changes to your project
+-------------------------
+When you added the SSO feature to your application, Teams Toolkit updated your project to support SSO:
+After you successfully added SSO into your project, Teams Toolkit will create and modify some files that helps you implement SSO feature.
+1. Create: 'aad.template.json' under 'templates/appPackage'
+   - The Azure Active Directory application manifest that is used to register the application with AAD.
+2. Modify: 'manifest.template.json' under 'templates/appPackage'
+   - An 'webApplicationInfo' object will be added into your Teams app manifest template. This field is required by Teams when enabling SSO.
+3. Create: 'Auth/tab'
+   - Reference code and a 'README.txt' file. These files are provided for reference. See below for more information.
+
+Update your code to add SSO
+-------------------------
+As described above, the Teams Toolkit generated some configuration to set up your application for SSO, but you need to update your application business logic to take advantage of the SSO feature as appropriate.
+
+1. Move 'GetUserProfile.razor' file under 'Auth/tab' to 'Components/'.
+   - 'GetUserProfile': This file implements a function that uses TeamsFx SDK to call Microsoft Graph API to get user info.
+2. Replace the following line: '<AddSSO />' with '<GetUserProfile />' to replace the 'AddSSO' component with 'GetUserProfile' component.
```

Comment:
Tell user which file to edit in this step?

Reply:
Updated, thx!